### PR TITLE
Set page title in the router

### PR DESCRIPTION
**Pull request by rudolfs**
This improves UX making it easier to find the tab that I'm looking for when I have multiple tabs open.

We can come up with a more elaborate structure, but for now this might be sufficient.

Screenshot 2023-07-11 at 17 30 59

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -93,7 +93,6 @@
       import { Buffer } from "buffer";
       window.Buffer = Buffer;
     </script>
-    <title>Radicle Interface</title>
```
Comment (geigerzaehler) [inline index.html:96]: I’d keep this and set it to “Radicle”. This way you’ll have a title until all the Javascript is loaded and the app has booted.

File: src/lib/router.ts [hunk 2]
```diff
@@ -112,10 +109,61 @@ async function navigate(
...
+      title.push("issues");
+    } else if (loadedRoute.params.view.resource === "patches") {
+      title.push(loadedRoute.params.project.name);
+      title.push("patches");
+    } else if (loadedRoute.params.view.resource === "patch") {
+      title.push(loadedRoute.params.view.params.loadedPatch.title);
+      title.push("patch");
+    } else {
+      utils.unreachable(loadedRoute.params.view);
+    }
```
Comment (geigerzaehler) [inline src/lib/router.ts:159]: To keep things modular I would move this code into a function in `app/views/project/router`. This way we keep the size of this code in check and we only have to change one file if we want to adjust the project routing.

File: index.html (hunk 1 above)
Comment (rudolfs) [reply index.html:96]: 2b7afe4917f870515910b624295a725559da4e48

File: src/lib/router.ts (hunk 2 above)
Comment (rudolfs) [reply src/lib/router.ts]: 3b1495657fa10d51157d6d2bcc76d8709490fee9

**Comment by cloudhead** [comment]
For the root (`/`) I'd just use "Radicle" as the title. For the rest I think it's good enough for now :+1:

**Review by geigerzaehler — APPROVED** [review]

**Pull request merged**